avocado/utils/pci.py

```diff
@@ -53,7 +53,7 @@ def get_pci_addresses():
     addresses = []
     cmd = "lspci -D"
     for line in process.system_output(cmd).splitlines():
-        if "PCI bridge" not in line:
+        if "PCI bridge" not in line and "peripheral" not in line:
```

Comment:
I see, if I understand (google searched) it correctly the 87d0 is basically a switch, therefor it applies for the category of bridge which you don't want to have in the results. Fair enough, only please use the full `System peripheral` to avoid possible clashes and please send it as a separate commit along with description (ideally include the `XXX:XX:XX.X System peripheral: PLX Technology, Inc. Device 87d0 (rev ca)` in the commit as an example of entry you want to ignore).